{{cookiecutter.app_name}}/test/supports/http.go

```diff
@@ -0,0 +1,24 @@
+package supports
```

Comment:
This package name is kind of awkward as it should be `support` (singular) and it does not help to know it's for testing. So how about changing the strategy again here 🙃 

![image](https://user-images.githubusercontent.com/696529/107914216-8c9eea80-6f94-11eb-9b25-78656a930bc7.png)

Basically having one `test` package:

```
├── test
│   ├── factories # -> to create test DB data e.g. createUser
│   ├── fixtures # -> for VCR, assets...
│   ├── http.go # -> helpers to make htttp request
│   └── test.go # -> to boostrap the tests
```

All support/helper can be split into separate in the directory `test`. In Go applications since the test files are next to each source file, the whole test folder can be dedicated to only helps.

Reply:
That sounds better and I totally agree with this structure. Fixed in 653c7cfec1cd035ff3440b2d163aeb39b8a2af8a